### ISSUE TITLE
Add additional LPs to ParameterExperiment scenario

### ISSUE THREAD

**Issue by cdummett** [vegaprotocol/vega-market-sim#305]
After the changes to the LP mechanics - the following network parameters now only have an effect on each LPs share of the liquidity fees (rather than the size of liquidity orders as before):

- `market.liquidity.minimum.probabilityOfTrading.lpOrders`
- `market.liquidity.probabilityOfTrading.tau.scaling`

Therefore multiple LPs are required in a scenario to see the effects of varying the above parameters.

The following LPs should be added to the `ParameterExperiment` scenario:

- An LP who commits liquidity just within liquidity-price-range bounds (added for future parameter experiments)
- An LP who commits liquidity just within price-monitoring limits (should be affected by varying `min_probability_of_trading`).
- An LP who commits liquidity mid-way between the best-bid or best-ask and the price monitoring bounds (should be affected by varying `tau_scaling`)

Most likely a `PriceSensitiveLiquidityProvider` agent should be created who can post liquidity at some configurable factor between the price-monitoring-bounds and the best-bid or best-ask.

**Comment by TomMcL** [comment]
I wouldn't call them `PriceSensitive` since the MMs are all somewhat price sensitive in that they quote around a target price :). But makes sense beyond that. 

With the quick investigation I did on Friday it was pretty easy to implement a first draft of this with a shaped market maker like:

```

wide_mm = ShapedMarketMaker(
    wallet_name="expon_wide",
    wallet_pass="expon",
    price_process_generator=iter(scenario.price_process),
    market_name=scenario.market_name + "_single_wide",
    asset_name=scenario.asset_name + "_single_wide",
    commitment_amount=scenario.lp_commitamount,
    market_decimal_places=scenario.market_decimal,
    asset_decimal_places=scenario.asset_decimal,
    tag='wide_mm',
    best_price_offset_fn=lambda a, b: (100, 100),
    shape_fn=lambda a, b: ([], []),
    liquidity_commitment_fn=lambda _: LiquidityProvision(
            amount=scenario.lp_commitamount,
            fee=0.00001,
            buy_specs=[
                [
                    "PEGGED_REFERENCE_MID",
                    5,
                    1,
                ]
            ],
            sell_specs=[
                [
                    "PEGGED_REFERENCE_MID",
                    5,
                    1,
                ]
            ],
    )
)
```

So something like that but more configurable sounds pretty good to me (also note that LPs are affected by the allowed quote distance on the market, which could be larger than or smaller than the price monitoring limits)

**Comment by cdummett** [comment]
Ah very nice thank you, will try repurpose the ShapedMarketMaker as much as possible going forwards then.

> (also note that LPs are affected by the allowed quote distance on the market, which could be larger than or smaller than the price monitoring limits)

Good point, I've added an LP committing close to the liquidity-price range bounds to the list (expected zero fees when the the price-monitoring bounds are tighter than the liquidity-price-range bounds). Will need to add some logging for all these different LPs as well at some point.

**Comment by cdummett** [comment]
Closed by #319
